response_operations_ui/templates/ce-seft-instrument.html

```diff
@@ -10,7 +10,6 @@
 
 {% block main %}
     {% include 'collection_exercise/ce-success-panel.html' %}
-    {% include 'collection_exercise/ce-handle-errors.html' %}
```

Comment:
No idea why we had 2 * {% include 'partials/error-box.html' %}  and a {% include 'collection_exercise/ce-handle-errors.html' %} On error I was getting multiple error boxes appearing, for this and other errors this page could generate